cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -564,24 +565,29 @@ func (impl *OSBuildJobImpl) Run(job worker.Job) error {
 				break
 			}
 
-			logWithId.Infof("[GCP] 🚀 Uploading image to: %s/%s", targetOptions.Bucket, targetOptions.Object)
+			bucket := targetOptions.Bucket
```

Comment:
I think that a nice addition would be erroring out if the bucket name isn't set in either the config, nor target options.